### ISSUE TITLE
[MAINTENANCE] Book - Pong Tutorial - Something wrong with SpriteSheetFormat in 0.13

### ISSUE THREAD

**Issue by sajattack** [amethyst/amethyst#2137]
## Description
The book tells you to set your Cargo.toml amethyst version to 0.13 and the provided pong_spritesheet.ron doesn't load correctly in 0.13.

```
[ERROR][amethyst_assets::storage] "renderer::SpriteSheet": Asset "texture/pong_spritesheet.ron" (handle id: Handle { id: 0 }) could not be loaded: Failed to load asset with name "texture/pong_spritesheet.ron"
[ERROR][amethyst_assets::progress] Error loading handle 0, renderer::SpriteSheet, with name texture/pong_spritesheet.ron: Failed to load asset with name "texture/pong_spritesheet.ron"
caused by: Failed to load asset with name "texture/pong_spritesheet.ron"
Error { inner: Inner { source: Some(Error { inner: Inner { source: Some(Error { inner: Inner { source: None, backtrace: None, error: LoadSpritesheetError } }), backtrace: None, error: Format("SPRITE_SHEET") } }), backtrace: None, error: Asset("texture/pong_spritesheet.ron") } }
caused by: Format "SPRITE_SHEET" could not load asset
Error { inner: Inner { source: Some(Error { inner: Inner { source: None, backtrace: None, error: LoadSpritesheetError } }), backtrace: None, error: Format("SPRITE_SHEET") } }
caused by: Failed to parse SpriteSheet
Error { inner: Inner { source: None, backtrace: None, error: LoadSpritesheetError } }
[ERROR][amethyst_assets::progress] Note: to handle the error, use a `Progress` other than `()`
[WARN][amethyst_assets::storage] Loading unnecessary asset. Handle 6 is unique
[ERROR][amethyst_assets::progress] Error loading handle 6, Mesh, with name texture/pong_spritesheet.png: Asset was loaded but no handle to it was saved.
caused by: Asset was loaded but no handle to it was saved.
Error { inner: Inner { source: None, backtrace: None, error: UnusedHandle } }
[ERROR][amethyst_assets::progress] Note: to handle the error, use a `Progress` other than `()`
```

What do you see that needs to be updated? Include a link to the code if possible.
Either change the book to tell you to use 0.14 or make the spritesheet compatible with 0.13.

**Comment by sajattack** [comment]
This seems to have happened to someone else too. https://github.com/amethyst/amethyst/issues/2034#issuecomment-560040851

**Comment by Giesch** [comment]
Changing the spritesheet ron linked in the book to be a plain tuple instead of a List(tuple) resolves the error on 0.13.
